src/components/Calculator.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
- [ ]  Please make sure you are not disabling eslint rules. These are here to help you write clean code.